src/main/kotlin/net/atos/zac/flowable/delegate/SuspendZaakDelegate.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.flowable.delegate
+
+import net.atos.zac.flowable.FlowableHelper
+import org.flowable.common.engine.api.delegate.Expression
+import org.flowable.engine.delegate.DelegateExecution
+import java.util.logging.Logger
+
+class SuspendZaakDelegate : AbstractDelegate() {
+    // Set by Flowable. Can be either FixedValue or JuelExpression
+    lateinit var aantalDagen: Expression
```

Comment:
Why in Dutch? Why not `numberOfDays`?

Reply:
And elsewhere.

Reply:
This one is part of the API IMHO, as it is visible in Flowable Designer. I think it would be easier for BPMN developers if all APIs are in Dutch. Furthermore this are parameters that can be modified by different role (not a developer) in future.